### PR TITLE
Weight estimator content updates (icons, text, data)

### PR DESCRIPTION
**Pull request by aileendds**
## Checklist

I have…

- [x] run the application locally (`bin/rails server`) and verified that my changes behave as expected.
- [x] run static code analysis (`bin/rubocop`) and vulnerability scan (`bin/brakeman`) against my changes.
- [x] run the test suite (`bin/rake spec`) and verified that all tests pass.
- [x] summarized below my changes and noted which issues (if any) this pull request fixes or addresses.
- [x] thoroughly outlined below the steps necessary to test my changes.
- [x] attached screenshots illustrating relevant behavior before and after my changes.

## Summary of Changes

This pull request…

- Update icons + text (https://github.com/deptofdefense/TRANSCOM/issues/28)
- Update data based on Lilly's changes in this Google sheet: https://docs.google.com/spreadsheets/d/1iyRqnq_Zed-sEmvYN_0xWP2V88iIE0BSKkwJ8GZ3CMM/edit#gid=0

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
2. git checkout weight-estimator-updates
3. set up development dependencies according to CONTRIBUTING.md,
4. run bin/rails db:drop db:create db:migrate db:seed,
5. run bin/rails server, and
6. load up http://localhost:3000/resources/weight-estimator in your Web browser of choice, check content

## Screenshots

screen shot 2017-10-25 at 10 35 11 am

### CODE REVIEW

**Comment by jamesatheyDDS** [comment]
Why are these not SVGs?

File: db/seeds/hhg_weights.json [hunk 1]
```diff
@@ -182,10 +182,6 @@
         "weight": 70
       },
       {
-        "name": "Typewriter",
```
Comment (jamesatheyDDS) [inline db/seeds/hhg_weights.json]: Typewriter LOL

File: db/seeds/hhg_weights.json [hunk 2]
```diff
@@ -694,10 +666,6 @@
         "weight": 420
       },
       {
-        "name": "Organ (Medium)",
```
Comment (jamesatheyDDS) [inline db/seeds/hhg_weights.json]: Not enough servicemembers have an organ to justify this, huh?

Comment (aileendds) [reply db/seeds/hhg_weights.json]: Yeah seems like a rare item

File: app/assets/images/icons/bedroom.svg [hunk 3]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="1200.000000pt" height="1200.000000pt" viewBox="0 0 1200.000000 1200.000000"
+ preserveAspectRatio="xMidYMid meet">
+<metadata>
+Created by potrace 1.15, written by Peter Selinger 2001-2017
```
Comment (aileendds) [inline app/assets/images/icons/bedroom.svg]: Hm the converter I used added this... Not sure if it's ok / I should use a different converter.

**Comment by jgarber623-gov** [comment]
Could you run the new PNGs through [svgo](https://github.com/svg/svgo) (the default options should be okay). It'll trim down a bunch of the editor-introduced cruft in the SVGs.

File: db/seeds/hhg_weights.json (hunk 1 above)
Comment (jgarber623-gov) [reply db/seeds/hhg_weights.json]: You'd be surprised where typewriters turn up… 😐

**Pull request merged**